network/network_common.go

```diff
@@ -188,8 +193,6 @@ func externalIpAddress() (string, error) {
 	return "", errors.New("not connected to the network")
 }
 
-type MacAddress struct{}
```

Comment:
Not used I suppose?

Reply:
Yes, and same thing for a few other similar types in the same file.

There are other types which are unused in windows files but after discussing with @derekwbrown he thought it was best to keep them (they are hand-made conversions from windows types), hence a few `//nolint:unused` (I guess I could have just commented them but we can change that later if we want to avoid any nolint).